policyengine_us/parameters/gov/states/oh/tax/income/exemptions/amount.yaml

```diff
@@ -0,0 +1,14 @@
+description: Ohio provides the following exemption credit amount for each exemption claimed.
+values:
+  2021-01-01: 20
+metadata:
+  unit: currency-USD
+  period: year
+  label: Ohio Exemption Credit amount
+  reference:
+    - title: Ohio Revised Code - Section 5747.022： Tax credits for exemptions
```

Comment:
add section

Reply:
The amount twenty appears in this link. I guess the section has been mentioned as 5747.022 already. Should I add something else?